reflection.md

```diff
@@ -6,4 +6,4 @@ on the following question:
 Is there anything that is still annoying or difficult about the operations we
 have in the library (i.e., flipping, rotating, and grayscaling)?
 
-_replace this text with your answer_
+I actually found it to be extremely convenient that instead of having to write out several lines of code, I was able to perform all three actions on the image within a single line. This, I believe, was because we no longer needed to save and and load the image into each function since the input and output type for each was a BufferedImage. However, the only modification I think could've been made would be to refactor the code since there are several repeated sections for each function, for instance, creating a new BufferedImage. Another thing I found to be inconvenient was that in order to load an image into the program, the user has to type out the entire path to it; I think it would be extremely convenient to thus create a function which would instead allow the user to simply write out the name of the image to import it.
```

Comment:
You did a great job in implementing the ```Square```,```Repeat``` and ```SleepIn``` function. Your response is mostly the same as mine. And the tests you added to test the 
```SleepIn``` function are complete and correct. 

For the picture program, you did an excellence job too! You modified the functions so that it pass in a ```BufferedImage``` type and return a ```BufferedImage``` type, and your implementation is just the same as mine. One thing you did better than me is that you even added three more methods modifying the picture. I actually learnt a lot from your coding. Especially, I learnt to blur an image, as you did in your Picture library. 

For your reflection, I agreed to most of your ideas. Yes, it is much more fluent to modify a image within a single line of code. But I think it would be more clearer and easier for the user if you separate those lines of code into three steps. Second, you talked about it was inconvenient to type out the entire path to load an image. Maybe you can try the function ```loadImage()``` and implement it like this:
```var picture = Picture.loadImage(resource("/image.png"))```
This code worked on my computer well.